content/en/docs/architecture/ci-operator.md

```diff
@@ -393,6 +393,7 @@ The `cluster_claim` below claims an OCP 4.7 cluster in AWS from a pool owned by
 ```yaml
 - as: e2e
   cluster_claim:
+    as: custom    # optional; if not specified, cluster claim release will be imported as `latest`
```

Comment:
Probably I should read the code of https://github.com/openshift/ci-docs/pull/155/files
Do I understand it correctly?
`as: custom`: `custom` will be imported to the namespace.
`as` is not specified: `latest` will be imported to the namespace.
Is it possible to import nothing?